less/shared.less

```diff
@@ -276,24 +280,15 @@ input.payment-type + label {
   padding: 6px 0;
   margin-left: 0;
   border-radius: 3px;
-  background-color: #EAEFF2;
   text-align: center;
   vertical-align: middle;
   cursor: pointer;
   white-space: nowrap;
   line-height: 1.42857;
-  transition: all 0.3s ease 0s;
```

Comment:
Old styles had transitions on hover, new flow did not. We handled this by resetting the transition to none in new flow.

Now that we're committing, instead of setting them to all -> none, let's just set not set it in the first place.